ReactAndroid/src/main/java/com/facebook/react/views/slider/ReactSlider.java

```diff
@@ -108,4 +129,76 @@ private void updateValue() {
   private int getTotalSteps() {
     return (int) Math.ceil((mMaxValue - mMinValue) / mStep);
   }
+
+  public void setThumbImage(ReadableMap source) {
+    String uri = source != null ? source.getString(PROP_ICON_URI) : null;
+    if (uri != null) {
+      if (uri.startsWith("http://") || uri.startsWith("https://")) {
```

Comment:
Looking at some code that does something similar we also check for file:// (see [this](https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/views/toolbar/ReactToolbar.java#L285))

The code to load drawables should probably be moved to some helper class as it seems to be copy pasted in a couple of places. -_- Let's leave this for another PR though.